jep/7/README.adoc

```diff
@@ -67,22 +69,68 @@ endif::[]
 
 == Abstract
 
-Due to its negative impact on core maintenance, distribution of the unmaintained `ruby-runtime` plugin will be suspended.
-Distribution of plugins depending on it will be suspended until their dependency is removed.
+Due to its negative impact on core maintenance, distribution of the unmaintained
+link:https://github.com/jenkinsci/ruby-runtime-plugin[Ruby Runtime plugin] and
+link:https://github.com/jenkinsci/jenkins.py[Jenkins Python SDK] will be suspended.
+Distribution of plugins depending on them will be suspended until their dependency is removed.
 
 
 == Specification
 
-`ruby-runtime` will be added to the `artifact-ignores` file of the update center generator so it is no longer available for download on update sites.
-// Perhaps: Add security warning?
+Ruby runtime and Python SDK will be added to the `artifact-ignores` file of the update center generator, so it is no longer available for download on update sites.
+A warning will be added to the update center so that all users of the plugins receive a warning in advance.
+The plugins will be also marked for adoption so that users can step up and take over maintenance without a 2 weeks waiting delay.
```

Comment:
is this something that we even want?  
how does it impact the CI system, the automatic secure releases (JEP-i-forget-the-number)?

I think this is mixing up the plugins that depend on the runtimes and the actual runtim plugins themselves?

Reply:
So the idea is to let contributors to quickly takeover the plugins and to migrate them to the supported runtimes.
No plan to recover the existing runtimes unless there is a strong commitment to maintain them by a large contributor

Reply:
so then I think this is unclear as written as "The plugins will be marked for adoption" includes the runtime ones.

```suggestion
Plugins depending on the either the Ruby runtime or the Python SDK will be also marked for adoption so that users can step up and take over maintenance without a 2 weeks waiting delay.
```

Reply:
```suggestion
Plugins depending on the either the Ruby runtime or the Python SDK will be also marked for adoption so that users can step up and take over maintenance (by switching to the regular Java runtime) without a 2 weeks waiting delay.
```